src/bp/core/routers/auth.ts

```diff
@@ -35,10 +43,14 @@ export class AuthRouter implements CustomRouter {
   }
 
   getAuthConfig = async () => {
+    const config = await this.configProvider.getBotpressConfig()
+    const strategy = _.get(config, 'pro.auth.strategy', 'basic')
```

Comment:
Do you actually need to provide 'basic' as default value since it's already the default config ?